kornia/morphology/dilation.py

```diff
@@ -0,0 +1,52 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class Dilate(nn.Module):
+
+    def __init__(self, se: torch.Tensor) -> None:
+        super().__init__()
+        self.se = se - 1
+        self.se_h, self.se_w = se.shape
+        self.pad = (self.se_h // 2, self.se_w // 2)
+
+        def se_to_mask(se: torch.Tensor) -> torch.Tensor:
```

Comment:
Better put se_to_mask as separate util function, which is called by both erode and dillate. Otherwise it is code duplication